style.css

```diff
@@ -1277,13 +1218,139 @@
     }
   }
 
+  .popup_window {
+    position: fixed;
+    top: 8%;
+    width: 80vw;
+    height: 80vh;
+    padding: 24px;
+    display: flex;
+    flex-direction: column;
+    background-color: #fff;
+    z-index: 225;
+  }
```

Comment:
- [ ] On the mobile version, kindly fix the spacings between the text and the `see live`, `see source`  buttons to match the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471&mode=design&t=eeb0Pc6Vb9CrUQao-0)

### Your design

<img width="329" alt="Screenshot 2023-07-02 at 00 23 11" src="https://github.com/lewisjn-arch/Mobile-Portfolio-setup/assets/64774936/d33154cf-bef0-4583-aba9-4c99c6577d62">

### Figma design

![Screenshot 2023-07-02 at 00 23 11 (2)](https://github.com/lewisjn-arch/Mobile-Portfolio-setup/assets/64774936/227d4396-14e9-4418-aebb-33ec1d8e524b)